x-pack/plugins/security_solution/public/common/components/endpoint/host_isolation/endpoint_host_isolation_status.tsx

```diff
@@ -136,9 +146,11 @@ export const EndpointHostIsolationStatus = memo<EndpointHostIsolationStatusProps
       dataTestSubj,
       getTestId,
       isIsolated,
-      isPendingStatuseDisabled,
+      isPendingStatusDisabled,
       pendingIsolate,
       pendingUnIsolate,
+      wasIsolating,
+      wasReleasing,
```

Comment:
this is actually not doing anything here because these will never change and thus `useMemo()`'s factory callback will never be executed if the `.current` value actually changes. Thats because `useRef` returns you the same object on every render.

If after my comment above, you decide to go with the `useRef()` approach, then these here should probably be:

```suggestion
      wasIsolating.current,
      wasReleasing.current,
```

Reply:
Yeah, this is unnecessary, will remove the dependencies.